test/built-ins/Temporal/PlainDateTime/minute-undefined.js

```diff
@@ -5,12 +5,19 @@
 esid: sec-temporal.plaindatetime
 description: Minute argument defaults to 0 if not given
 features: [Temporal]
+includes: [temporalHelpers.js]
 ---*/
 
 const args = [2000, 5, 2, 12];
 
 const explicit = new Temporal.PlainDateTime(...args, undefined);
-assert.sameValue(explicit.minute, 0, "minute default argument");
+
+TemporalHelpers.assertPlainDateTime(
+  explicit,
+  2000, 5, "M05", 2, 12, 0, 0, 0, 0, 0,
+  "minute default argument (argument present)"
+);
 
 const implicit = new Temporal.PlainDateTime(...args);
-assert.sameValue(implicit.minute, 0, "minute default argument");
+
+TemporalHelpers.assertPlainTimesEqual(explicit, implicit, "minute default argument (argument missing)");
```

Comment:
In any case, this should compare PlainDateTimes, not PlainTimes

Reply:
right, that's clearly a blooper -- I've adopted your suggestion to use `assertPlainDateTime` rather than `assertPlainDateTimesEqual`, thereby fixing this particular issue.